chroma-manager/Makefile

```diff
@@ -228,18 +235,17 @@ requirements.txt: ../chroma-dependencies/Makefile ../chroma-dependencies/*/Makef
 	fi
 	echo "jenkins_fold:end:Make requirements.txt"
 
-download: requirements.txt
-	echo "jenkins_fold:start:Install Python requirements"
-	pip install -r requirements.txt
-	echo "jenkins_fold:end:Install Python requirements"
+download: install_requirements
```

Comment:
@tanabarr Yes, you are correct.  Sorry, I confused the concern about this target with the others.

But what is this `download` target doing?  It has no recipe, so why does it need to have the requirements installed?  Why is it even needed?